net/lighttpd/files/lighttpd.conf

```diff
@@ -24,9 +24,6 @@ server.feature-flags       += ("server.graceful-shutdown-timeout" => 5)
 #server.errorlog-use-syslog  = "enable"
 #server.network-backend      = "writev"
 
-### Use IPv6 if available
```

Comment:
Btw, this was removed and I could not see why, which results that you are making more changes than saying in the commit itself.